lib/ethon/easy/informations.rb

```diff
@@ -46,6 +46,12 @@ module Informations
         # start until the name resolution was completed.
         :namelookup_time => :double,
 
+        # Return the time, in seconds, it took for all redirection steps
+        # include name lookup, connect, pretransfer and transfer before the
+        # final transaction was started. time_redirect shows the complete
+        # execution time for multiple redirections. (Added in 7.12.3)
```

Comment:
It should be coming from here: http://curl.haxx.se/libcurl/c/curl_easy_getinfo.html#CURLINFOREDIRECTTIME. But it seems to be wrong. Maybe it has changed in the libcurl docs.